UnityClient/Assets/Scripts/SwipeCamera.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SwipeCamera : MonoBehaviour
+{
+
+    private Touch _initTouch = new Touch();
```

Comment:
nit: Maybe initialise this in `Awake()` instead? I'm not a big fan of direct field initialisation due to the fact the CLR doesn't guarantee initialisation order.